src/workflows/default.js

```diff
@@ -1,28 +1,9 @@
 import * as run from "./steps";
+import { rebaseUpdateLogCommitTagRelease } from "./shared";
 
-export default [
+export default [].concat( [
```

Comment:
With ES6 there is another way to do `[].concat`. See following example...

```js
const oneThroughThree = [ 1, 2, 3 ];
const sevenThroughNine = [ 7, 8, 9 ];

console.log(
  [].concat( oneThroughThree, [ 4, 5, 6 ], sevenThroughNine )
); // ES5

console.log(
  [ ...oneThroughThree, 4, 5, 6, ...sevenThroughNine ]
); // ES6
```

http://jsbin.com/nerejumeti/edit?js,console,output